doc/whats_new/v0.8.rst

```diff
@@ -8,6 +8,8 @@ Changelog
 
 `ramp-database`
 ...............
+- Add possibility to download the leaderboards (private and public) as .csv
```

Comment:
sleeping over it I think the word download is not ideal as it's CLI option. I would replace the word download by export in the API and the CLI option.